boards/arduino-mkr-common/Makefile.include

```diff
@@ -17,7 +17,7 @@ ifeq ($(PROGRAMMER),jlink)
 else
   # by default, we use BOSSA to flash this board and take into account the
```

Comment:
here and below: comment needs updating

Reply:
I don't think so, BOSSA is the default flash tool (for these boards).

Reply:
I updated it accordingly.

Reply:
Ok, I was looking at fixed version but the initial comment is still there ;)